bitbucket-pipelines.yml

```diff
@@ -0,0 +1,53 @@
+image: 
+  name: aligent/shopify-cli:latest
+
+definitions:
+  steps:
+    - step: &theme-check
+        name: Theme Check
+        script:
+          - echo "Starting THEME CHECK..."
+          - ruby -v && node -v && npm -v && shopify version
+          - find . -exec ls -ld --time-style=long-iso {} \; | awk '{print $6, $7, $8, $9}'
+          - echo -e "\nRunning shopify theme check --fail-level=$FAIL_LEVEL\n"; shopify theme check --fail-level=$FAIL_LEVEL
+
+    - step: &deploy
+        name: Deploy Theme
+        script:
+          - echo "Starting THEME CHECK..."
+          - ruby -v && node -v && npm -v && shopify version
+          - find . -exec ls -ld --time-style=long-iso {} \; | awk '{print $6, $7, $8, $9}'
+          - echo -e "\nRunning shopify theme check --fail-level=$FAIL_LEVEL\n"; shopify theme check --fail-level=$FAIL_LEVEL
+          - echo -e "DEPLOYING to $BITBUCKET_DEPLOYMENT_ENVIRONMENT...\n\n$ASCII_LABEL\n"
+          - echo -e "shopify theme push --store=$SHOPIFY_STORE --password=$SHOPIFY_CLI_ACCESS_TOKEN --theme=$THEME_ID $THEME_PUSH_ARGS\n"
+          - shopify theme push --store="$SHOPIFY_STORE" --password="$SHOPIFY_CLI_ACCESS_TOKEN" --theme="$THEME_ID" $THEME_PUSH_ARGS
+
+pipelines:
+  branches:
+    feature/*:
+      - step: *theme-check
+
+    staging:
+      - step:
+          <<: *deploy
+          deployment: Staging
+
+    production:
+      - step: 
+          script:
+            - echo "Start of production deployment. Press DEPLOY button when you are ready."
+      - step:
+          <<: *deploy
+          trigger: manual
+          deployment: Production
+
+  pull-requests:
+    staging:
```

Comment:
I don't think these two are required. Might be best to remove these and move the theme-check here instead